src/app/authentication/repositories/forgotPasswordRepository.go

```diff
@@ -21,7 +19,6 @@ func NewForgotPasswordRepository(db *gorm.DB) ForgotPasswordRepository {
 }
 
 func (repository *userDBRepository) VerifyAndUpdatePassword(email string, pancardNumber string, newPassword string) error {
-	// Update the password for the user if found
 	result := repository.DB.Model(&dbModels.Users{}).
 		Where("email = ? AND pan_card = ?", email, pancardNumber).
```

Comment:
added column names in genericConstants and use.
const group is already present in src>>constants>>constants.go just add the column name there.

// Column Name Constants
const (